core-java-modules/core-java/src/main/java/com/baeldung/java/hexagonal/DefaultUserService.java

```diff
@@ -0,0 +1,38 @@
+package com.baeldung.java.hexagonal;
+
+import java.util.Collection;
+import java.util.Optional;
+
+/**
+ *  Default implementation of user service 
+ */
+public class DefaultUserService implements UserService{
```

Comment:
indentation.

Reply:
These changes are tab indented and should not be in this PR. Please revert them.

Reply:
@sinsuren and @ashleyfrieze 

Thanks for the review. These files are part of my first demo article and accidentally part of this PR. I have now removed the complete package. I'll be careful and verify the commits part of the PR.